Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -776,7 +776,7 @@ public void RaiseSourceDetected(IMixedRealityInputSource source, IMixedRealityCo
             HandleEvent(sourceStateEventData, OnSourceDetectedEventHandler);
         }
 
-        private static readonly ExecuteEvents.EventFunction<IMixedRealitySourceStateHandler> OnSourceDetectedEventHandler =
+        public static readonly ExecuteEvents.EventFunction<IMixedRealitySourceStateHandler> OnSourceDetectedEventHandler =
```

Comment:
It would be great if we could get this working without flipping these to `public`. These definitions should really stay private to the input system and not directly referenced outside it.